hasher-matcher-actioner/hmalib/common/configs/fetcher.py

```diff
@@ -26,3 +26,17 @@ class ThreatExchangeConfig(HMAConfig):
     def privacy_group_id(self) -> str:
         """TE Configs are keyed by their privacy group ID"""
         return self.name
+
+
+@dataclass
+class NonThreatExchangeConfig(HMAConfig):
+    """
+    Config for the NonThreatExchange datasets(i.e. StopNCII hashes)
+    """
+
+    fetcher_active: bool
```

Comment:
looks like the majority of these fields are shared with `ThreatExchangeConfig` is it possible to use a shared base class?